apps/evm/app/(landing)/components/Hero.tsx

```diff
@@ -101,8 +101,8 @@ export const Hero: FC = () => {
                 type="OUTPUT"
                 className="p-3 bg-white dark:bg-slate-800 rounded-xl"
                 value={valueIndex >= 0 ? VALUES[valueIndex].value1 : ''}
-                onChange={() => {}}
-                onSelect={() => {}}
+                onChange={() => undefined}
```

Comment:
Might be a better idea to just make the change to the component which would allow for these handlers to be undefined.

Reply:
Yeah makes sense, was just quickly fixing a lint error